lib/src/main/kotlin/com/swmansion/starknet/account/Account.kt

```diff
@@ -188,18 +188,22 @@ interface Account {
      * @param calldata constructor calldata for the contract deployment
      * @param salt salt used to calculate address of the new contract
      * @param forFeeEstimate when set to `true`, it changes the version to `2^128+version` so the signed transaction can only be used for fee estimation
+     * @param l1ResourceBounds L1 resource bounds for the transaction
+     * @param l2ResourceBounds L2 resource bounds for the transaction
      * @return signed deploy account payload
      */
     fun signDeployAccountV3(
         classHash: Felt,
         calldata: Calldata,
         salt: Felt,
         l1ResourceBounds: ResourceBounds,
+        l2ResourceBounds: ResourceBounds,
```

Comment:
Definitely, makes sense.

Reply:
> I just don't fully like the fact that now the param of type `ResourceBoundsMapping` is named `resourceBounds` 😆 , but looks like it's because of [this](https://github.com/starkware-libs/starknet-specs/blob/8cf463b79ba1dd876f67c7f637e5ea48beb07b5b/api/starknet_api_openrpc.json#L2114), so we can leave it.

That was the idea, but also the fact that `resourceBoundsMapping` doesn't seem like a good argument name from UX point of view.

I think such naming convention also makes sense in this particular case:
- `ResourceBounds` - `l1ResourceBounds`, `l2ResourceBounds`
- `ResourceBoundsMapping` - `resourceBounds`